### PR TITLE
[ULMS-1971] Add API to create a replica of webinar

### PR DESCRIPTION
**Pull request by mgrachev**

### CODE REVIEW

File: src/app/api/v1/authz/mod.rs [hunk 1]
```diff
@@ -366,16 +368,19 @@ async fn substitute_class(
...
             {
                 None => Ok(()),
-                Some(AuthzClass { id }) => {
+                Some(AuthzClass {
+                    id,
+                    original_class_id,
+                }) => {
                     *obj = "classrooms".into();
-                    *set_id = id;
+                    *set_id = original_class_id.unwrap_or(id);
```
Comment (manifest) [inline src/app/api/v1/authz/mod.rs:376]: Why do we substitute class id with the original class id?

File: src/app/api/v1/webinar/convert.rs [hunk 2]
```diff
@@ -138,12 +139,19 @@ async fn do_convert(
...
             .context("Failed to acquire transaction")
             .error(AppErrorKind::DbQueryFailed)?;
+
         let webinar = query
             .execute(&mut txn)
             .await
             .context("Failed to find recording")
             .error(AppErrorKind::DbQueryFailed)?;
 
+        crate::db::class::UpdateContentIdQuery::new(webinar.id(), webinar.id().to_string())
```
[the remark this answers is not in the document]
Comment (manifest) [reply src/app/api/v1/webinar/convert.rs:149]: Agred on setting `content_id=NULL` for new webinars.

File: migrations/20220726091936_add_content_id_to_class.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+ALTER TABLE class ADD COLUMN content_id text;
+CREATE INDEX class_content_id ON class (content_id);
```
Comment (manifest) [inline migrations/20220726091936_add_content_id_to_class.sql:2]: Do we care about records `content_id=NULL`  presented in the index?

```sql
CREATE INDEX class_content_id ON class (content_id) WHERE content_id IS NOT NULL;
```

Comment (manifest) [reply migrations/20220726091936_add_content_id_to_class.sql:2]: By the way, where are we going to use it?

Comment (mgrachev) [reply migrations/20220726091936_add_content_id_to_class.sql]: Fixed.

> By the way, where are we going to use it?
https://github.com/foxford/dispatcher/pull/152/files#diff-28c02084e7909f1babe2844d503aa4b9d7625368789caed3c8a634e39f824b39R105

Comment (manifest) [reply migrations/20220726091936_add_content_id_to_class.sql]: Shouldn't `content_id` be `NULL` for the new classes?

Comment (mgrachev) [reply migrations/20220726091936_add_content_id_to_class.sql]: Yes, you're right.

**Review by manifest — APPROVED** [review]

**Pull request merged**